### ISSUE TITLE
🐛Show Correct Error Message if the Diagram Cannot be Parsed

### ISSUE THREAD

**Issue by Vyperus** [process-engine/bpmn-studio#1333]
#### Describe your issue

If your try to deploy a diagram that cannot be parsed by the PE a generic Error "The diagram cannot be parsed" is thrown.
We get the correct error messages from the PE and we should display them correctly in BPMN-Studio to help the user as much was we can.

#### How can others reproduce the issue?

1. Start BPMN-Studio.
2. Try to deploy this diagram:
https://github.com/5minds/dog-food-klub/issues/14

#### Possible solution

Show the correct error messages.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by robpal94** [comment]
The question here is, what does the error message qualifies as _correct_? 
If a Process cannot be parsed, the backend answers with a Message like: 

```
Unprocessable Entity Error - The XML for Diagram ... could not be parsed. Cyclic Timer Events are currently not supported
```

Here, the message is way to long to display it in the small Toaster, but what else should the User see in this case?

**Comment by Vyperus** [comment]
My plan ist to use the bpmn-js-linter in someway to cover this.

**Comment by Vyperus** [comment]
Closed with #1383
